snap/info.go

```diff
@@ -144,6 +144,7 @@ type SideInfo struct {
 // Info provides information about snaps.
 type Info struct {
 	SuggestedName string
+	InstanceKey   string
```

Comment:
When/where is this set?

Reply:
It will be set based on data from `overlord/snapstate.SnapState`.